cvmfs/sync_mediator.cc

```diff
@@ -254,8 +256,12 @@ bool SyncMediator::Commit(manifest::Manifest *manifest) {
     LogCvmfs(kLogPublish, kLogStdout, "");
   }
 
+  if (!params_->print_changeset) {
```

Comment:
See above, to be replaced by `reporter_->CommitReport()`

Reply:
The point of the `CommitReport()` would be rather that it takes care of all the printing.  I.e. inside `CommitReport` it would figure out whether or not an additional linebreak is needed so that we don't need to look at `params_->print_changeset` at all.